### ISSUE TITLE
Alternative for systemd

### ISSUE THREAD

**Issue by DRY411S** [Bisa/factorio-init#91]
I was able to follow your install instructions perfectly on a CentOS 7 box, but am now trying it on an Amazon Web Services LMI instance.

This does not have systemd, and so this step in your Readme does not work.

Can you advise of alternatives please, or does your script rely on systemd?

**Comment by Xiretza** [comment]
I guess just running the script normally doesn't work (i.e. `/opt/factorio-init/factorio start`)?

**Comment by DRY411S** [comment]
It wasn't, which is why I posted this. But I'm using 0.14.12 on AWS and the script is broken.  (I was using 0.13 stable on CentOS).

Applying the fixes from issue #87  means that the script works. I also changed the SERVICE_NAME in the config file to lose the upper case 'F'. (i.e the 'service' is the name of the executable). I don't know whether case sensitivity is an issue.

Amazon Web Services on their LMI Linux image require you to 'init' processes by adding scripts in through their web portal. Therefore I can start factorio automatically on reboot using that method rather than needing systemd and enabling through systemctl.

Therefore closing this issue.
